examples/manifold/plot_mds.py

```diff
@@ -13,6 +13,12 @@
 # Authors: The scikit-learn developers
 # SPDX-License-Identifier: BSD-3-Clause
 
+# %%
+# Dataset preparation
+# -------------------
+#
+# We start by uniformly generating 20 points in a 2D space.
+
```

Comment:
```rst
The reconstructed points using the metric MDS and non metric MDS are slightly
shifted to avoid overlapping.
```
Is that true ? I don't see where the shifting is done. I would remove that sentence if no longer relevant.

Reply:
Hmm. Good point. Removed this comment.